gpii/node_modules/solutionsRegistry/src/js/filtered-settings-middleware.js

```diff
@@ -0,0 +1,174 @@
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+fluid.require("%kettle");
+fluid.require("%gpii-json-schema");
+
+// Add the ability to require JSON5 files.
+require("json5/lib/register");
+
+// Require universal so that we can resolve paths to it and any sub-modules.
+require("../../../../../");
+
+require("./filter-schema-settings");
+require("./lib/get-settings-schema");
+
+fluid.registerNamespace("gpii.universal.solutionsRegistry.kettle.request.http");
```

Comment:
I appreciate the thinking behind this namespace but I think it has gone a little too far. I think something like gpii.universal.solutionsRegistry.requestValidation will be sufficient

Reply:
Replaced throughout.